### PR TITLE
will have /api/topics endpoint available

### PR DESCRIPTION
**Pull request by SimoneDessi**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const express = require("express")
+const getTopics = require("./controllers/controller_core")
+
+const app = express()
+
+app.use(express.json())
```
Comment (johnhutcheon) [inline app.js]: Hi Simone, we wouldn't need this line of code yet. Please ensure the code you're submitting is only relevant to the ticket you're currently submitting. Please remove this until it's required.

File: controllers/controller_core.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+
+const getTopics = (req, res) => {
+  selectTopics()
+  .then((topics) =>
+  {
+    res.status(200).send({ topics })
+  }).catch((err) => {
+    console.log(err)
+  })
```
Comment (johnhutcheon) [inline controllers/controller_core.js]: Do any of the tests you have written currently require this catch block to be in your code?

File: __tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+});
+describe("GET /api/topics", () => {
+  test("return 200 status, should return an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics[0]).toHaveProperty("slug");
+        expect(body.topics[0]).toHaveProperty
+        ("description")
```
Comment (johnhutcheon) [inline __tests__/app.test.js]: We are only checking the first element in the array here. Could we please refactor the test so we check every element? We'd also look to check the length of the array when we do this. 👍

Comment (Poonam-raj) [reply __tests__/app.test.js]: Hey Simone, this still needs actioning. As you're expecting multiple items back we need to make sure the test is checking the whole array (and that the right number of items are coming back) -> please refactor 🚀

**Review by Poonam-raj — CHANGES_REQUESTED** [review]
Heya Simone, 

Just need to see that test checking the whole array not just the first item.

Thanks,
Poonam :D

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Hi Simone! Nice work so far! 
I'm just going to request an amendment to your testing, please see my comment for more info. 👍 
I've also noted that you've named your controller and model as 'core', which I assume is related to the instructions that we've provided? I can understand why you've done this, however, to someone reading your repo who has not seen the instructions it would be slightly confusing, so could you please rename the files to just 'model' and 'controller'?

**Comment by johnhutcheon** [comment]
Thanks for making the changes Simone! All good to merge 🙌

**Pull request merged**